pwa-devdocs/src/frequently-asked-questions/index.md

```diff
@@ -75,6 +75,22 @@ To let the storefront query a specific store view in Magento you need to add the
 
     -   Add `headers:{ Store: YOUR_STORE_CODE }`
 
+## Image component loads wrongly sized images from the srcSet
+
+When you are using the `<Image/>` component from `venia-ui` you need to make sure to have changed the following values:
+
+-   [packages/venia-ui/lib/util/images.js][]
+
+    -   Change `DEFAULT_WIDTH_TO_HEIGHT_RATIO` to match your image ratio.
+    -   Change the values in the `imageWidths` mapping to better fit your dimensions.
+
+-   [packages/venia-ui/lib/components/gallery/item.js][]
+
+    -   Make sure to supply the `<Image />` component with a `widths` prop as shown in this example.
+
+{: .bs-callout .bs-callout-info}
+_**Note:** The native device emulator in Chrome will give you incorrect values, it's best to resize the viewport._
```

Comment:
```suggestion
_**Note:** For testing, resize the viewport manually instead of using the native device emulator in Chrome, which gives incorrect values._
```